### DISCUSSION TITLE
I want the URL to be auto-generated by dividing the folder.

### DISCUSSION THREAD

**Discussion by JangCool** [nuxt/nuxt#18299] [Questions] [answered]
I want the URL to be auto-generated by dividing the folder.
Is there a way?

ex)
Currently available methods are:

pages/Cache.vue => http://localhost:3000/cache
pages/Was.vue => http://localhost:3000/was

I want this :

```
pages/ 
          cache/
                    Index.vue => http://localhost:3000/cache
           was/
                    Index( OR Was).vue => http://localhost:3000/was
```

**Comment by danielroe** [comment, marked as answer]
I'm not sure what you mean. Have you tried using lower-case `index.vue`? `~/pages/cache/index.vue` should match `/cache`.

**Reply by JangCool** [reply]
@danielroe  I'm sorry. your answer is correct Thank you. I wanted to set the folder name to the root path. I confirmed that the index.vue file name should be written in lowercase.

**Reply by JangCool** [reply]
@danielroe  If so, should all .vue files to pages folder be written in lowercase? ? Should components folder be uppercase and pages folder should be lowercase?

I can make a naming rule, but I wonder how you use it.
